uuv_manipulators_kinematics/scripts/kinematics_service.py

```diff
@@ -120,7 +120,7 @@ def solve_ik(self, req):
         result_ik = self._arm_interface.inverse_kinematics(pos, orientation)
 
         if result_ik is not None:
-            for i, name in zip(range(len(self.joint_names)), self.joint_names):
+            for i, name in zip(list(range(len(self.joint_names))), self.joint_names):
```

Comment:
Let's leave this for now. It has to be a list to work with zip and I doubt there are millions of joint_names...

Reply:
oops, my mistake. zip will work with a range object. Not a big deal but I think slightly better/more readable to use enumerate. I'll push this now.